packages/storage/src/indexed-db/config.ts

```diff
@@ -2,4 +2,4 @@
  * The version number for the IndexedDB schema. This version number is used to manage
  * database upgrades and ensure that the correct schema version is applied.
  */
-export const IDB_VERSION = 44;
+export const IDB_VERSION = 45;
```

Comment:
To clarify, this will cause mainnet users must sync again after updating the extension in chrome store. Should we already consider indexed-db migrations without data loss?

Reply:
Think we are ok for now, syncing is very fast with the low block height at the moment